src/lib/components/NotificationList.svelte

```diff
@@ -7,6 +7,19 @@
 
   let notificationsList: INotification[] = [];
 
+  enum Position {
+    TOP_RIGHT = 'top-right',
+    TOP_LEFT = 'top-left',
+    BOTTOM_RIGHT = 'bottom-right',
+    BOTTOM_LEFT = 'bottom-left'
+  }
```

Comment:
```suggestion
  enum Position {
    TopRight = 'top_right',
    TopLeft = 'top_left',
    BottomRight = 'bottom_right',
    BottomLeft = 'bottom_left'
  }
```

TypeScript uses Capiltalized Camel Case for Enums, refer to: https://www.typescriptlang.org/docs/handbook/enums.html.

Also, please use Snake Case over Kebab Case for strings on enums, so they are variable friendly.